JSDemos/Demos/Toolbar/Multiline/Vue/App.vue

```diff
@@ -0,0 +1,492 @@
+        </template>
+
+        <template #menuSeparatorTemplate>
+          <div class="menu-separator"/>
+        </template>
+
+        <template #colorpicker="{ data }">
+          <div class="custom-color-picker">
+            <i
+              v-for="(itemColor, i) in data"
+              :key="i"
+              :class="
+                itemColor
+                  ? 'color dx-icon dx-icon-square'
+                  : 'color dx-icon dx-icon-square dx-theme-text-color'
+              "
+              :style="{ color: itemColor }"
+              @click="onColorClick(itemColor)"
+            />
+          </div>
+        </template>
+
+        <template #fontSizeTemplate="{ data }">
+          <div :style="{ fontSize: data.size + 'px' }">{{ data.text }}</div>
+        </template>
+
+        <template #textAlignTemplate>
+          <DxButtonGroup
+            key-expr="alignment"
+            styling-mode="outlined"
+            :items="getTextAlignItems(false)"
+            :selected-item-keys="textAlign"
+            @item-click="onTextAlignItemClick"
+          />
+        </template>
+
+        <template #textAlignMenuTemplate>
+          <DxButtonGroup
+            :items="getTextAlignItems(true)"
+            display-expr="text"
+            :selected-item-keys="textAlign"
+            key-expr="alignment"
+            styling-mode="outlined"
+            @item-click="onTextAlignItemClick"
+          />
+        </template>
+      </DxToolbar>
+    </DxResizable>
+
+    <div class="options-container">
+      <div class="caption">Options</div>
+
+      <DxRadioGroup
+        layout="horizontal"
+        :items="reactivePriorities"
+        :value="priority"
+        :on-value-changed="onChangeHandler"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import DxCheckBox from 'devextreme-vue/check-box';
+import DxToolbar, { DxItem } from 'devextreme-vue/toolbar';
+import DxButtonGroup from 'devextreme-vue/button-group';
+import DxResizable from 'devextreme-vue/resizable';
+import DxDropDownButton from 'devextreme-vue/drop-down-button';
+import DxDateBox from 'devextreme-vue/date-box';
+import DxSelectBox from 'devextreme-vue/select-box';
+import DxRadioGroup from 'devextreme-vue/radio-group';
+import notify from 'devextreme/ui/notify';
+
+import {
+  colors,
+  fontSizes,
+  lineHeights,
+  fonts,
+  fontStyles,
+  textAligns,
+  listTypes,
+} from './data.js';
+import 'devextreme/ui/select_box';
+
+const priorities = ['Multiline mode', 'Singleline mode'];
+
+export default {
+  components: {
+    DxCheckBox,
+    DxToolbar,
+    DxButtonGroup,
+    DxItem,
+    DxResizable,
+    DxRadioGroup,
+    DxDropDownButton,
+    DxDateBox,
+    DxSelectBox,
+  },
+
+  data() {
+    return {
+      multiline: true,
+      colorPicker: null,
+      color: null,
+      priority: priorities[0],
+      lineHeight: lineHeights[1].lineHeight,
+      textAlign: [textAligns[0].alignment],
+    };
+  },
+
+  computed: {
+    reactiveColors() {
```

Comment:
why reactive? :)

Reply:
Removed in https://github.com/DevExpress/devextreme-demos/pull/2004/commits/5ba43954aa5e53c1a0c2a8596281ef2bf941d78c